src/xcode/ENA/ENA/Source/Services/PPAnalyticsCollector/Model/PPADataType.swift

```diff
@@ -49,6 +49,25 @@ enum PPATestResultMetadata {
 }
 
 enum PPAKeySubmissionMetadata {
+	case create(KeySubmissionMetadata, SubmissionTestType)
+	case submitted(Bool, CoronaTestType)
+	case submittedInBackground(Bool, CoronaTestType)
+	case submittedAfterCancel(Bool, CoronaTestType)
+	case submittedAfterSymptomFlow(Bool, CoronaTestType)
+	case submittedWithTeletan(Bool, CoronaTestType)
+	case submittedWithCheckins(Bool?, CoronaTestType)
+	case lastSubmissionFlowScreen(LastSubmissionFlowScreen?, CoronaTestType)
+	case advancedConsentGiven(Bool, CoronaTestType)
+	case submittedAfterRapidAntigenTest(CoronaTestType)
+	case setHoursSinceTestResult(CoronaTestType)
+	case setHoursSinceTestRegistration(CoronaTestType)
+	case setDaysSinceMostRecentDateAtENFRiskLevelAtTestRegistration(SubmissionTestType)
+	case setHoursSinceENFHighRiskWarningAtTestRegistration(SubmissionTestType)
+	case setDaysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration(SubmissionTestType)
+	case setHoursSinceCheckinHighRiskWarningAtTestRegistration(SubmissionTestType)
+}
+
+enum SrsPPAKeySubmissionMetadata {
```

Comment:
is this even used anywhere?

Reply:
i used it first then decided to compbine the srs with the normal submission metadata, will remove it now